database/viewer.sql

```diff
@@ -76,7 +76,19 @@ create or replace view viewer.viewer_object_map as
             --st_astext(pand.pand_centroid) as pand_centroid,
             -- ipv st_collect() zou st_extent() ook kunnen
             (select st_astext(st_centroid(st_collect(the_geom))) from wfs."Polygon" where "DBK_ID" = vo.id) as pand_centroid,
-            st_astext(selectiekader.the_geom) as selectiekader,
+            
+            -- Omdat een selectiekader kan bestaan uit meerdere gebieden de left join vervangen voor een subquery.
+            --st_astext(selectiekader.the_geom) AS selectiekader,
+            --st_astext(st_centroid(selectiekader.the_geom)) AS selectiekader_centroid,
+            (select st_astext(st_extent("Gebied".the_geom))
```

Comment:
I assume st_extent is used here because st_collect/st_union do not work for some reason and for select_area later on it does work, maybe add a comment why?
It seems to me this change means that objects will be selected for an incident when the incident coordinate is in the Gebied extent but not intersecting the Gebied itself...?